SourceCode/raytracer/raytracer/CRTSceneLoader.cpp

```diff
@@ -1,11 +1,16 @@
+#pragma warning( disable : 4365 )
+
 #include <fstream>
 
+#include <memory>
 #include "json.hpp"
 
+#include "Animation.h"
```

Comment:
A bit strange to see the Animation involved in the parsing of the scene

Reply:
My plan was initially to add animations to the .crtscene file, but later scratched that. It was much easier to script my animations in an `onTick` function from the Engine loop